test/images/no-snat-test-proxy/main.go

```diff
@@ -40,20 +43,36 @@ func NewMasqTestProxy() *MasqTestProxy {
 	}
 }
 
+// NewTestProxyCommand creates a *cobra.Command object with default parameters
+func NewTestProxyCommand() *cobra.Command {
+	m := NewMasqTestProxy()
+	cmd := &cobra.Command{
+		Use:  "MasqTestProxy",
+		Long: `MasqTestProxy`,
+		Run: func(cmd *cobra.Command, args []string) {
+			verflag.PrintAndExitIfRequested()
+			m.Run()
+		},
+	}
+	m.AddFlags(pflag.CommandLine)
```

Comment:
@liggitt this should be `cmd.Flags()`, right?

Reply:
yes. in general, we shouldn't be moving toward global flagsets anywhere

Reply:
@hanxiaoshuai I will update and squash into your commit.